ncl/ncl_entries/trigonometric_functions.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Probably a broader discussion, but part of the "grab and go" appeal was that it was right up top and was our "recommendation". With having multiple sections with multiple options each, I think we're missing that.</p><p> </p><p>What would folks think about doing something this just once at the top?</p><p> </p><h2>Grab and Go</h2>

<pre class="ql-syntax" spellcheck="false">import numpy as np

rads = [0, np.pi/2, np.pi, 3*np.pi/2, 2*np.pi]

s = np.sin(rads)
c = np.cos(rads)
t = np.tan(rads)
as = np.asin(rads)
...

print(f"sin: {s}")
...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I have it split up since on the NCL index page they are separate functions and having separate sections lets the index hyperlink to the section of the notebook that is relevant

Reply:
I suppose I can see your point there. I'm curious what other folks think.

@jukent, I believe the grab and go sections were your idea originally, do you have any thoughts? Also @kafitzgerald, any opinions?

In a case where we might end up having different grab and go sections (here or elsewhere), I still think we should lean towards only having one recommendation in the grab and go unless there's a specific reason not to (like, use `x` if you're doing `y` workflow, `b` works better with `c` use cases).

Reply:
I'm a little split between a) merge as-is and see how that looks and feels along with everything else b) an inclination to try and keep this a bit more simple and aligned with for instance the days_in_month NCL section grab and go.

I do think it's maybe a bit verbose as is and we may want to reorder the structure at some point, but I also don't know that we have a super clear idea of what will work best at this point and I think that might be ok.  There could be some value in just getting a few examples in and then re-evaluating what will work best. 
 Especially this early on I don't know that we can expect to get everything "right" and templated out in a way that is robust.

That said, it can be hard to come back and refactor / rearrange, but if we're intentional about it - I'm not too worried.

Reply:
Okay, I'll drop my concern for the purpose of merging this PR, we can talk about this particular point in a broader conversation.